README.md

```diff
@@ -132,7 +115,15 @@ lobid-organisations is a web app implemented with Play to serve the JSON-LD cont
 On start up, the web app will transform the data and build an Elasticsearch index from the output of the transformation. These steps can be triggered separately using HTTP POST when the application is up and running. Before building the index, the application will check for the minimum size of the transformation output. This is done to prevent building up an index that only contains part of the available data or no data at all (e.g. if something goes wrong during the transformation, the result may be an empty file). This minimum size threshold is specified in `conf/application.conf`. In addition, during transformation updates of the Sigel data can be fetched --- you can specify the date (i.e. the date of the dump, e.g. 2013-06-01) from which want the updates to start in `conf/application.conf`. Updates will be downloaded from this date on
 until today.
 
-Run the Play application: `~/activator-1.3.10-minimal/bin/activator run`
+Run the Play application:
+
+Set the variable to the home folder, not the path of your JAVA installation:
+e.g.: `export JAVA_HOME="/usr"`
+
+
+- `sbt clean`
+- `sbt --java-home $JAVA_HOME stage`
+- `JAVA_OPTS="$JAVA_OPTS -XX:+ExitOnOutOfMemoryError" ./target/universal/stage/bin/lobid-organisations -Dhttp.port=9000 -no-version-check`
```

Comment:
These steps are mentioned before. Only difference is the port number. You could reference the already existing recipe and explain why to change the port.

Reply:
One does not need the `--java-home` reference for staging or the other additional parameters for startin the web app locally.